src/HeaderWrapper.js

```diff
@@ -117,31 +139,38 @@ export const HeaderWrapper = ({
         transform: `translateX(${-translateX}px)`
       }}
     >
-      {mappedColumns.current.map((el, index) => (
-        <HeaderCellWrapper
-          isEmpty={index === emptyColumn.current}
-          onMouseDown={handleMouseDown}
-          width={el.width}
-          text={el.headerName}
-          onChangeWidth={onChangeWidth}
-          index={index}
-        />
-      ))}
-      {isMoving && (
-        <div
-          style={{
-            position: "absolute",
-            left: `${startCoord.x}px`,
-            top: `${startCoord.y}px`,
-            transform: `translateX(${mouseMove}px)`,
-            width: `${movingColumnData.current.width}px`,
-            outline: "1px solid black",
-            height: `${startCoord.height}px`
-          }}
-        >
-          {movingColumnData.current.headerName}
-        </div>
-      )}
+      <div style={{width: `${tempWidth}px`}}>
+        {draggableColumns.map((el, index) => (
+          <HeaderCellWrapper
+            key={index}
+            isEmpty={index === emptyColumn.current}
+            onMouseDown={handleMouseDown}
+            headerRef={headerRef}
+            fullWidth={fullWidth}
+            width={el.width}
+            text={el.headerName}
+            onChangeWidth={onChangeWidth}
+            onResize={handleColumnResize}
+            index={index}
+          />
+        ))}
+
+        {isMoving && (
+          <div
```

Comment:
How about call it MovingHeaderCell and move `position: "absolute"` and `outline: "1px solid black"` there?